events/serializers.py

```diff
@@ -18,6 +19,11 @@ def user_is_lecturer_in_event(event, user_id):
     return user_id in lecturers_ids
 
 
+def user_payed_for_event(user, event):
+    payments = PaymentsRepository().get_payment_for_user_and_event(user, event).exists()
```

Comment:
we can simplify it to 
```
return PaymentsRepository().get_payment_for_user_and_event(user, event).exists()
```

Reply:
Yes we could, but this way it is easier to debug, when you first assign it to a variable it is possible to check exactly what value it will return